contracts/VaultV3.vy

```diff
@@ -509,9 +537,9 @@ def _issue_shares(shares: uint256, recipient: address):
 @internal
 def _issue_shares_for_amount(amount: uint256, recipient: address) -> uint256:
     """
-    Issues shares that are worth 'amount' in the underlying token (asset)
+    Issues shares that are worth 'amount' in the underlying token (asset).
     WARNING: this takes into account that any new assets have been summed 
-    to total_assets (otherwise pps will go down)
+    to total_assets (otherwise pps will go down).
```

Comment:
is it OK to assume reader knows what "pps" means? or should we change all "pps" references to "pricePerShare"?

Reply:
I think its probably self explanatory given context if you are reading code in a vault. Though just to make sure i added a " (pps) " comment to the actual `pricePerShare` function so that a simple ctrl f "pps" will be able to find an explanation.

d083af79f77b844ae0ce106b7699cd00849294a6